src/os/ui/file/csv/abstractcsvexporter.js

```diff
@@ -97,6 +97,34 @@ os.ui.file.csv.AbstractCSVExporter.prototype.process = function() {
 os.ui.file.csv.AbstractCSVExporter.prototype.processItem = goog.abstractMethod;
 
 
+/**
+ * Get whether to always include WKT geometry in the export
+ * @return {boolean} If true, WKT geometry will always be included in the export
+ */
+os.ui.file.csv.AbstractCSVExporter.prototype.getAlwaysIncludeWkt = goog.abstractMethod;
```

Comment:
These functions are specific to geospatial/temporal CSV export, so I would leave them solely on `plugin.file.csv.CSVExporter`. Your CSV export UI can assume its exporter is a `plugin.file.csv.CSVExporter`. 

If other CSV exporters have a need for these functions in the future, we could extract the API to an interface and use `os.implements` to detect support from each exporter. That's more work than necessary at the moment.